packages/@aws-cdk/applet-js/test/test-nonstack.yaml

```diff
@@ -1,4 +1,6 @@
-#!/usr/bin/env cdk-applet-js
-applet: ./test-applet:NoStackApplet
-argument: this should be reflected in the template description
+Applets:
```

Comment:
Any chance I can convince you to use lower case letters for the schema?

Reply:
You can, and I'm not liking this either.

But then let's not mimic the CloudFormation schema at all. Otherwise it's just going to be confusing, don't you think?

Reply:
Both [Azure Resource Manager](https://docs.microsoft.com/en-us/azure/azure-resource-manager/resource-group-authoring-templates) and [Google Cloud Deployment Manager](https://cloud.google.com/deployment-manager/docs/configuration/syntax-reference) use very similar syntaxes. This is basically the standard-de-facto for desired state configuration.

Reply:
Alrighty